Sources/Common/Extensions/DateExtensions.swift

```diff
@@ -18,18 +18,34 @@ extension Date {
         return formatter.string(from: self)
     }
 
-    func addMinutes(_ minutes: Int) -> Date {
+    func add(_ value: Double, _ unit: Calendar.Component) -> Date {
```

Comment:
seems unnecessary as just calls `add(int(value))` so the cast could be performed at the call site instead